docs/guides/CODE_OF_CONDUCT_GUIDE.md

```diff
@@ -3,12 +3,21 @@
 Promote a welcoming and safe environment for discussions and contributions.
 
 > [!TIP]
-> More information is available from [GitHub documentation](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project).
+> More information is available
+>
+from [GitHub documentation](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project).
```

Comment:
Could you please extract the link so that the linebreak is more natural?

Reply:
Fixed.